### ISSUE TITLE
[swagger-codegen-cli] Can't disable docs or tests generation

### ISSUE THREAD

**Issue by Klemsus** [swagger-api/swagger-codegen#8522]
##### Description

Disabling doc or test generations (like in an example in README.md) is not working
```
>java -Dapis -DapiTests=false -jar swagger-codegen-cli.jar generate -l java -o rest -i ot15-rest-openapi-agents-1.1.8.yml --library retrofit2
[Thread-1] WARN io.swagger.codegen.v3.ignore.CodegenIgnoreProcessor - Output directory does not exist, or is inaccessible. No file (.swagger-codegen-ignore) will be evaluated.
[Thread-1] WARN io.swagger.codegen.v3.generators.DefaultCodegenConfig - Empty operationId found for path: get /presence/contacts. Renamed to auto-generated operationId: presenceContactsGet
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\main\java\io\swagger\client\api\AcdccdApi.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\test\java\io\swagger\client\api\AcdccdApiTest.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\docs\\AcdccdApi.md
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\main\java\io\swagger\client\api\PresenceApi.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\test\java\io\swagger\client\api\PresenceApiTest.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\docs\\PresenceApi.md
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\main\java\io\swagger\client\api\TelephonyApi.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\test\java\io\swagger\client\api\TelephonyApiTest.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\docs\\TelephonyApi.md
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\main\java\io\swagger\client\api\UserApi.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\src\test\java\io\swagger\client\api\UserApiTest.java
[Thread-1] INFO io.swagger.codegen.v3.AbstractGenerator - writing file C:\Sergey\Work\Projects\swagger-codegen\modules\swagger-codegen-cli\rest\docs\\UserApi.md
```
Same for docs (and for modelDocs and modelTests also)

##### Swagger-codegen version

3.0.0-SNAPSHOT

##### Command line used for generation

`java -Dapis -DapiTests=false -jar swagger-codegen-cli.jar generate -l java -o rest -i ot15-rest-openapi-agents-1.1.8.yml --library retrofit2`

##### Steps to reproduce

Just try to generate without tests or docs

**Comment by Klemsus** [comment]
`<generateApiDocumentation>false</generateApiDocumentation>`
and others in swagger-codegen-maven-plugin are also not working

**Comment by mfvitale** [comment]
Same problem for me.

**Comment by olekst** [comment]
> `<generateApiDocumentation>false</generateApiDocumentation>`
> and others in swagger-codegen-maven-plugin are also not working

Using version 3.0.0 of swagger-codegen-maven-plugin worked for me.

**Comment by frantuma** [comment]
fixed in #8980
